docs/api.rst

```diff
@@ -95,3 +95,6 @@ Aggregation
    aggregation.ocean.antarctic
    aggregation.ocean.ice_shelves
    aggregation.ocean.transport
+
+   aggregation.seaice
+   aggregation.seaice.arctic
```

Comment:
Nice. And yes, I like the idea of having the seaice folder with these regions in it.